src/ui/component/claimPreview/view.jsx

```diff
@@ -93,6 +102,14 @@ function ClaimPreview(props: Props) {
         (outpoint.txid === claim.txid && outpoint.nout === claim.nout)
     );
   }
+  // block stream claims
+  if (claim && !shouldHide && !showUserBlocked && blockedChannelUris.length && signingChannel) {
+    shouldHide = blockedChannelUris.some(blockedUri => blockedUri === signingChannel.permanent_url);
+  }
+  // block channel claims if we can't control for them in claim search
+  if (claim && isChannel && !shouldHide && !showUserBlocked && blockedChannelUris.length && isChannel) {
+    shouldHide = blockedChannelUris.some(blockedUri => blockedUri === claim.permanent_url);
```

Comment:
Why do we need this if we use `not_channel_ids`?

Reply:
Oh, I guess on the search/related content lists. Can you add a comment saying it's explicitly for that?